File: app/src/main/java/zone/com/zview/Activity/ViewActivity.java
package zone.com.zview.Activity;

import and.base.activity.BaseActivity;
import zone.com.zview.view.SinView;

/**
 * Created by fuzhipeng on 2016/11/18.
 */

public class ViewActivity extends BaseActivity {
    @Override
    public void setContentView() {
        setContentView(new SinView(this));
    }

    @Override
    public void findIDs() {

    }

    @Override
    public void initData() {

    }

    @Override
    public void setListener() {

    }
}

File: app/src/main/java/zone/com/zview/Activity/ViewActivity.kt
package zone.com.zview.Activity

import com.zone.lib.base.controller.activity.BaseFeatureActivity

import zone.com.zview.view.SinView

/**
 * Created by fuzhipeng on 2016/11/18.
 */

class ViewActivity : BaseFeatureActivity() {

    override fun setContentView() {
        setContentView(SinView(this))
    }

    override fun initData() {

    }

    override fun setListener() {

    }
}
